### ISSUE TITLE
Add a setting to make links non-clickable

### ISSUE THREAD

**Issue by GilianJuice** [a85/POSTMan-Chrome-Extension#13066]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

The problem is that Postman makes all links in requests and responses clickable by default. When I click such a link, Postman opens a new tab inside Postman with the link in the URL field. I don't understand why it works like this by default, it's very rare that I actually want to open a link inside Postman itself. When the link is an URL to an external thing, such as an image or a file, I would want to open it in my browser, not in Postman. When it's a link to the API, to a different endpoint or to load the next page of results for example, opening a new tab without headers, authentication, body etc is kinda pointless. It's easier to duplicate my request and then change the URL.

Sometimes I want to copy a part of a link from an API response, such as a product ID or whatever, that is really hard to do when Postman wants to open that link in a new tab whenever I click in it to select (part of) the URL. Often I just accidentally click a link when I want to click something else, then Postman adds a new tab and I have to close it again. When closing the tab, Postman doesn't even switch back to the tab I came from, it switches to the most right tab that I have active.

### Describe the solution you'd like

I understand that some people maybe do like links to be clickable, so my suggestion is to make a setting for it so people can decide for themselves what they want. The default value for this setting can be so that current behavior of Postman doesn't change. People who want to change the behavior can then do it with that setting.

### Describe alternatives you've considered

I found that I can right-click a link in Postman and choose "Copy" to copy the entire link. This helps in some cases, but not if I want to copy a part of the URL or accidentally click it.

**Comment by maksymlumen** [comment]
Yeah, this is annoying. Postman does that when you even trying to copy selected text with a link in it.

**Comment by dnlpva** [comment]
This would be a nice feature to have.

**Comment by ed-mak** [comment]
Looking for updates on this. 

In our company, we have a lot of api responses that return back responses with urls. We never actually want to click on or use the urls - it's just part of the data. So when we're clicking around the output, it is very annoying to just randomly click on a link (from a json response) and have it open. Additionally even more painfully annoying, we also have json fields with forward slashes "/" and those get detected as links too. 

There is little safe space to click in the postman output. This is one of the more cumbersome aspects of using postman. I imagine that disabling the link detection would also improve performance and responsiveness.
